wp-content/plugins/vf-hero-container-lp/group_vf_wp_hero_lp.json

```diff
@@ -42,104 +39,102 @@
+            "default_value": "",
+            "placeholder": "",
+            "prepend": "",
+            "append": "",
+            "maxlength": ""
         },
         {
-            "key": "field_5f9a7bdb636f0",
-            "label": "Hero Heading",
-            "name": "vf_hero_heading",
+            "key": "field_5fc8c1246eb93",
+            "label": "Subheading",
+            "name": "vf_hero_subheading",
             "type": "text",
-            "instructions": "Use the placeholder <code> %1$s<\/code> for the blog name.",
+            "instructions": "",
             "required": 0,
             "conditional_logic": 0,
             "wrapper": {
                 "width": "50",
                 "class": "",
                 "id": ""
             },
-            "default_value": "About the %1$s",
+            "default_value": "",
             "placeholder": "",
             "prepend": "",
             "append": "",
             "maxlength": ""
         },
         {
-            "key": "field_5f9a7bdb63781",
-            "label": "Hero Introduction",
+            "key": "field_5fc8b06b0535a",
+            "label": "Text",
             "name": "vf_hero_text",
-            "type": "text",
-            "instructions": "Leave blank to use introduction from the Content Hub.",
+            "type": "wysiwyg",
+            "instructions": "",
             "required": 0,
             "conditional_logic": 0,
             "wrapper": {
-                "width": "50",
+                "width": "",
                 "class": "",
                 "id": ""
             },
             "default_value": "",
-            "placeholder": "",
-            "prepend": "",
-            "append": "",
-            "maxlength": ""
+            "tabs": "all",
+            "toolbar": "basic",
+            "media_upload": 0,
+            "delay": 0
         },
         {
-            "key": "field_5f9a7bdb63811",
-            "label": "Hero Link",
-            "name": "vf_hero_link",
+            "key": "field_5fc8b06b053eb",
+            "label": "Additional heading 1",
+            "name": "vf_hero_additional_heading_1",
             "type": "link",
-            "instructions": "(Optional) The link will appear after the Hero Introduction text. Example: 'Read more'",
+            "instructions": "Optional",
             "required": 0,
             "conditional_logic": 0,
             "wrapper": {
-                "width": "",
+                "width": "50",
                 "class": "",
                 "id": ""
             },
             "return_format": "array"
         },
         {
-            "key": "field_5f9a7bdb638a1",
-            "label": "Hero Image",
-            "name": "vf_hero_image",
-            "type": "image",
-            "instructions": "Recommended size: 1224×348 (larger images are cropped).",
+            "key": "field_5fc8b06b0547c",
+            "label": "Additional heading 2",
+            "name": "vf_hero_additional_heading_2",
```

Comment:
2?

Reply:
Ah - so this, as mentioned, might change anyhow … but for `vf-eleventy` we just allow people to pass in the HTML string - so it can be one thing, two things, one with a link etc.

Of course - that relies on the person editing the content to know HTML -- so this probably works better.

Reply:
There are two separate fields for those two:
![Screenshot 2020-12-03 at 12 46 16](https://user-images.githubusercontent.com/54020899/101014095-999b6880-3565-11eb-9d2c-7a4010ebea84.png)